bodhi/server/services/releases.py

```diff
@@ -332,3 +337,75 @@ def save_release(request):
     request.db.flush()
 
     return r
+
+
+@critpath_packages.get(permission='edit', renderer='json',
```

Comment:
This actually might be the issue - the ```edit``` permission might be why the 403 is happening that I wrote about earlier. Unauthenticated users don't get any permissions, so you should be able to just drop that argument here.